content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -372,7 +372,7 @@ running in a second cluster. Before you begin:
 
     The configurations above will result in all traffic in `cluster1` for
     `httpbin.bar.global` on *any port* to be routed to the endpoint
-    `<IPofCluster2IngressGateway>:15443` over a mutual TLS connection.
+    `CLUSTER2_GW_ADDR:15443` over a mutual TLS connection.
```

Comment:
do you need to add $ sign to wrap this? like `$CLUSTER2_GW_ADDR`

Reply:
I'm fine either way. This is being evaluating in bash so left the ${} it off as I thought it might be easier to read. But did have it there for one read through and would be fine putting it back.

Reply:
yeah, since I see we use it in other places.

Reply:
Update made